README.md

```diff
@@ -261,15 +261,8 @@ package managers:
 - `gem.txt`: run-time dependencies installed by gem.
 - `npm.txt`: run-time dependencies installed by npm.
 - `pip.txt`: a normal [pip `requirements.txt`][] file, for run-time
-  dependencies too.
-
-All of these files will install separately each line. See this example
```

Comment:
I specifically had them install separately, because identifying issues in the installation when they are grouped together is damned near impossible. Seems we've already lost it though, so I'll make do.